pkg/master/master.go

```diff
@@ -671,7 +671,7 @@ func (m *Master) api_v1beta2() *apiserver.APIGroupVersion {
 func (m *Master) api_v1beta3() *apiserver.APIGroupVersion {
 	storage := make(map[string]rest.Storage)
 	for k, v := range m.storage {
-		if k == "minions" {
+		if k == "minions" || k == "minions/status" {
```

Comment:
It would probably take a regex - I don't want minions or any of their sub resources.

----- Original Message -----
> > @@ -671,7 +671,7 @@ func (m *Master) api_v1beta2()
> > *apiserver.APIGroupVersion {
> >  func (m *Master) api_v1beta3() *apiserver.APIGroupVersion {
> >  	storage := make(map[string]rest.Storage)
> >  	for k, v := range m.storage {
> > -		if k == "minions" {
> > +		if k == "minions" || k == "minions/status" {
> 
> Is there a general problem to be solved with strings.Contains(k, "/")?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/7969/files#r29967768